nimare/meta/cbma/mkda.py

```diff
@@ -523,20 +528,22 @@ def _fit(self, dataset1, dataset2):
         del pFgA_sign, pAgU
 
         maps = {
+            "z_desc-uniformity": pAgF_z,
+            "z_desc-association": pFgA_z,
+            "chi2_desc-uniformity": pAgF_chi2_vals,
+            "chi2_desc-association": pFgA_chi2_vals,
+            "p_desc-uniformity": pAgF_p_vals,
+            "p_desc-association": pFgA_p_vals,
             "prob_desc-A": pA,
             "prob_desc-AgF": pAgF,
             "prob_desc-FgA": pFgA,
-            ("prob_desc-AgF_given_pF=%0.2f" % self.prior): pAgF_prior,
-            ("prob_desc-FgA_given_pF=%0.2f" % self.prior): pFgA_prior,
-            "z_desc-consistency": pAgF_z,
-            "z_desc-specificity": pFgA_z,
-            "chi2_desc-consistency": pAgF_chi2_vals,
-            "chi2_desc-specificity": pFgA_chi2_vals,
-            "p_desc-consistency": pAgF_p_vals,
-            "p_desc-specificity": pFgA_p_vals,
         }
-        description = self._generate_description()
 
+        if self.prior is not None:
```

Comment:
```suggestion
        if self.prior:
```
with the current code, if `self.prior` is set to false it will still run the prior.
Taking anything that is truthy or false-y is also a bit weird, but a little more intuitive imo